test/contracts/CorkConfig.ts

```diff
@@ -3,28 +3,62 @@ import {
 } from "@nomicfoundation/hardhat-toolbox-viem/network-helpers";
 import { expect } from "chai";
 import hre from "hardhat";
+import { parseEther, parseUnits, zeroAddress } from "viem";
 import * as helper from "../helper/TestHelper";
 
 describe("CorkConfig", function () {
   let defaultSigner: any;
   let secondSigner: any;
   let signers: any;
 
+  let expiryTime: any;
```

Comment:
please add types to this, like the new uni v2 integration

Reply:
The way i would recommend this is by using Utility types from typescript, couple of it you might find useful is Awaited and ReturnType. please read [this](https://www.typescriptlang.org/docs/handbook/utility-types.html).

Reply:
Done : Added types for variables in CorkConfig.ts file and and types are already added in other test files through uniswap v2 PR so just focused on types in config test file

Reply:
Nice, please resolve the conflicts below. i recommend pulling from main first on your IDE and resolve the conflicts there. Don't resolve the conflicts through the Github built-in editor

Reply:
Fixed merge conflicts. Thanks

Reply:
There's another merge conflict caused by the freeze method PR, please fix them

Reply:
Merge conflict resolved